frontend/src/lib/components/LemonCheckbox/LemonCheckbox.scss

```diff
@@ -0,0 +1,52 @@
+.LemonCheckbox {
+    display: flex;
+    width: fit-content;
+    input {
+        appearance: none;
+        width: 0;
+        height: 0;
+    }
+    label {
+        display: flex;
+        align-items: center;
+    }
+}
+
+.LemonCheckbox__box {
+    transition: border 200ms ease, background 200ms ease;
+    background: var(--bg-light);
+    border: 1px solid var(--border-dark);
+    border-radius: var(--radius);
+    --fill: var(--primary);
```

Comment:
just curious, what does --fill do here?

Reply:
Oops, this was leftover from a previous approach where `color` was set with a CSS var, that turned out to be cumbersome. Removed.